### DISCUSSION TITLE
Organization Profile Page Views

### DISCUSSION THREAD

**Discussion by antonkomarev** [yhype/yhype-feedback#8] [Ideas]
We can add Organization Profile views counter the same way as User Profile Views. Is it required?
I want to get feedback to see how many people really want this to try to find the time to implement.

**Comment by hatamiarash7** [comment]
At the moment, Every Organization can have a Readme repo ( like normal users ), For example, The official Github account :

![Screenshot 2022-04-10 at 14-26-24 GitHub](https://user-images.githubusercontent.com/16325641/162612800-6cd9fca9-bf36-407b-ba0b-1720f8342abc.jpg)

I think it is very good to have the same data in company ( organization ) profiles

**Reply by turian** [reply]
I would love this too because I lead several open-source organizations like

https://github.com/hearbenchmark

https://github.com/torchsynth

**Comment by MistressPlague** [comment]
+1

**Comment by andrey-helldar** [comment]
I think it's a useful feature to see how many real people are opening the page.

**Comment by antonkomarev** [comment]
This feature is in progress, but not active. Need to refactor a lot of stuff and re-parse all accounts data.
